Source/Installer/zh-Hant.lproj/MainMenu.xib

```diff
@@ -171,7 +171,6 @@ Gw
         <customObject id="494" customClass="AppDelegate">
             <connections>
                 <outlet property="cancelButton" destination="592" id="710"/>
```

Comment:
the `@IBOutlet` connection to `actionButton` is missing in the zh-Hant version of the xib.

Reply:
Oh, rats. Thanks for the catch.

Fixed in https://github.com/shuang886/McBopomofo/commit/353702c4b58529807582ae8c085e2e1fe0b5291a